src/components/switch/index.tsx

```diff
@@ -14,7 +15,16 @@ export interface SwitcherProps extends React.HTMLAttributes<HTMLElement> {
 
 export const Switcher: React.FC<SwitcherProps> = React.forwardRef<HTMLElement, SwitcherProps>(
   (
-    { id: idProp, checked = false, setChecked, height, width, className, wrapperProps, ...props }: SwitcherProps,
+    {
+      id: idProp,
+      checked = false,
+      setChecked,
+      height = 24,
+      width = 48,
```

Comment:
else you can add property `size` which has a proportion of 2:1 (2 is a width and 1 is a height) or another one (according to the design system)

it's useful during development and shorter ))